src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    private final Random random = new Random();
+
+    public Ball getRandomBall() {
+        int maxValueOfRandomDigit = 100;

```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/ec04c0c3-f391-4d3c-80fb-47a19afbe7e9)

Reply:
What's the problem with writing all the comments at once? this looks like a joke on my work.